scripts/build-sunshine.sh

```diff
@@ -116,7 +111,7 @@ then
 	case $SUNSHINE_PACKAGE_BUILD in
 		ON)
 			echo "Downloading package to: $BUILD_DIR/$CONTAINER_NAME.$SUNSHINE_PACKAGE_EXTENSION"
-			docker cp $CONTAINER_NAME:/root/sunshine-build/package-deb/sunshine.$SUNSHINE_PACKAGE_EXTENSION "$BUILD_DIR/$CONTAINER_NAME.$SUNSHINE_PACKAGE_EXTENSION"
+			docker cp $CONTAINER_NAME:/root/sunshine-build/package-$SUNSHINE_PACKAGE_EXTENSION/sunshine.$SUNSHINE_PACKAGE_EXTENSION "$BUILD_DIR/$CONTAINER_NAME.$SUNSHINE_PACKAGE_EXTENSION"
```

Comment:
We need to make the rpm build here:

`/root/sunshine-build/package-rpm/sunshine.rpm`

Is that something simple to change? Even if just copying the rpm file to that location?

Reply:
Can I add a `-d` to `gen-rpm` to retain compatability with running the script out of container?

Reply:
I think that's fine... Then in `build_private.sh` it will become `./gen-rpm -d` ?

Reply:
Done.